deploy/k8s/charts/trustification/values.yaml

```diff
@@ -212,4 +213,13 @@ modules:
     image: {}
     rust: {}
 
+  report:
+    enabled: true
+    ingress: {}
+    image: {}
+    enabled_auth: false
+    # Here's an example showing how to set up authentication.
```

Comment:
Why not allow the user to provide an existing `Secret`? We are using the approach in different parts of the chart.